Emitron/Emitron/UI/Library/Filtering/FiltersView.swift

```diff
@@ -70,7 +70,7 @@ struct FiltersView: View {
       
       HStack {
         
-        MainButtonView(title: "Clear All", type: .secondary(withArrow: false)) {
+        MainButtonView(title: "Reset filters", type: .secondary(withArrow: false)) {
```

Comment:
Nit - filters should be capitalised

```suggestion
        MainButtonView(title: "Reset Filters", type: .secondary(withArrow: false)) {
```